clients/apps/web/src/components/Dashboard/Account/Topbar.tsx

```diff
@@ -1,3 +1,4 @@
+import { Visibility } from '@/../../../packages/polarkit/src/api/client/models/Visibility'
```

Comment:
What happened here? 😂 Crazy import path!

Reply:
Ouch. Auto-generated by vscode. Will fix.

Reply:
Bah. 
<img width="599" alt="image" src="https://github.com/polarsource/polar/assets/374824/15bb845a-c0d7-4331-b713-f70cd9f0cbca">

Reply:
Fixed in #527